### PR TITLE
Support later versions of tenacity

### PR DESCRIPTION
**Pull request by tobias-urdin**
This adds support for later version of the
tenacity library and unpins the dependency.

This depends on the tooz [1] change or it
will fail.

[1] https://review.opendev.org/c/openstack/tooz/+/829412

### CODE REVIEW

File: setup.cfg
Comment (jd) [inline setup.cfg:42]: You might want to leave the min version, that should not change.

**Review by jd — CHANGES_REQUESTED** [review]

File: setup.cfg
Comment (tobias-urdin) [reply setup.cfg:42]: I will fix this.

**Comment by mrunge** [comment]
This looks good to me.

**Comment by tobias-urdin** [comment]
This is waiting for deprecation of etcd3 driver in Tooz [1] and then the fix to Tooz in [2] and a new release of Tooz.

[1] https://review.opendev.org/c/openstack/tooz/+/833107/1
[2] https://review.opendev.org/c/openstack/tooz/+/829412

**Comment by tobias-urdin** [comment]
This is waiting for a new release of tooz

**Comment by tobias-urdin** [comment]
tooz 2.11.0 is released which should unblock this

**Pull request merged**
